tests/testSetup.js

```diff
@@ -0,0 +1,30 @@
+const db = require("../db.js");
+var fs = require("fs");
+const path = require("path");
+
+const testData = fs
+  .readFileSync(path.resolve(__dirname, "./testData.sql"))
+  .toString();
+
+async function commonBeforeAll() {
+  await db.query(testData);
+}
+
+async function commonBeforeEach() {
+  await db.query("BEGIN");
+}
+
+async function commonAfterEach() {
+  await db.query("ROLLBACK");
+}
+
+async function commonAfterAll() {
```

Comment:
nit - I would rename these functions to what they do. I think it will read a bit better in the test files:

`beforeEach(commonBeforeEach);` vs `beforeEach(beginTransaction);`
`afterEach(commonAfterEach);` vs `beforeEach(rollbackTransaction);`

At some point, we will have to change setting up the db to be in every test as we do more of the create/update/delete  functionality

Reply:
Okay, I'll change the names. As for the jwt's, I was planning on adding that logic in my next pull request when I set up the actual login endpoint.